elyra/util/operation_parser.py

```diff
@@ -14,66 +14,60 @@
 # limitations under the License.
 #
 
-import asyncio
 import json
 import nbformat
-import os
 import re
 
 from abc import ABC, abstractmethod
 from jupyter_server.base.handlers import APIHandler
 from tornado import web
-from traitlets.config import SingletonConfigurable
-from typing import Any, Type, TypeVar, List
+from typing import Any, Type, TypeVar, List, Dict
 from ..util.http import HttpErrorMixin
 
 
 class OperationParserHandler(HttpErrorMixin, APIHandler):
     """Handler to expose REST API to parse envs from a File"""
 
     @web.authenticated
-    async def get(self):
+    async def post(self):
         msg_json = dict(title="Operation not supported.")
         self.write(msg_json)
         self.flush()
 
     @web.authenticated
-    async def post(self, *args, **kwargs):
-        payload = self.get_json_body()
+    async def get(self, path=''):
+        path = path or ''
 
-        self.log.debug("Parsing environmental variables from notebook: %s", payload['file_path'])
-        self.log.debug("JSON payload: %s", json.dumps(payload, sort_keys=True, indent=2, separators=(',', ': ')))
+        # TODO: Query and verify file type
+        # type = self.get_query_argument('type', default=None)
```

Comment:
I think we should always infer the operation from the path - even if it isn't a file-based operation (so 'path' is a path to the operation).